### DISCUSSION TITLE
License

### DISCUSSION THREAD

**Discussion by jesperkdab** [f35f22fan/Ods2#9] [General]
Hi,
I see you changed the license from QODS to GPL only.
This prevent me from switching (and contributing) to QODS2.

Was that an intentional change?

Cheers
Jesper.

**Comment by f35f22fan** [comment]
I didn't think anyone cares (frankly I don't even remember "changing" the
license), what other license do you need? MIT? If so I'll add it some time
later, for now I just don't have motivation to bother. To me the license is
a superficial thing as I never really cared what this or that license says
since I'm not working for any organization.

**Reply by jesperkdab** [reply]
Yes MIT would be perfect.

**Comment by f35f22fan** [comment]
MIT license added.

**Reply by jesperkdab** [reply]
Excellent, thx
